app.py

```diff
@@ -156,7 +154,7 @@
                     style={'border-width': '0'},
 
                     ################ The magic happens here
-                    srcDoc= Plotter.pm_linechart("Vancouver Kitsilano", pms = [2.5, 10], height = 250).to_html()
+                    srcDoc= Plotter.pm_linechart("Vancouver Kitsilano", pms = [2.5, 10], height = 250, width = 300).to_html()
```

Comment:
Is this supposed to have "Vancouver Kitsilano" because it is an outdated location name?